etc/kayobe/networks.yml

```diff
@@ -191,11 +194,20 @@ inspection_net_inspection_allocation_pool_end: "10.{{ inspection_net_vlan }}.102
 # inspection_net_mtu:
 # inspection_net_routes:
 
+# Cambridge ilab network IP information.
+ilab_net_vlan: 60
+ilab_net_cidr: "10.{{ ilab_net_vlan }}.0.0/16"
+ilab_net_routes:
+  # Default route onto the Cambridge ilab network
+  - cidr: "0.0.0.0/0"
+    gateway: "10.60.255.1"
```

Comment:
I wasn't 100% on this one, but I think this is correct.

Reply:
For reference this is what we have on the seed at the moment:

[doug@cumulus-seed cumulus-kayobe-config]$ sudo ip route show
default via 10.60.255.1 dev em1.60